python_helpers/rx_udp.py

```diff
@@ -31,5 +32,17 @@
     print("python - awaiting new connection...")
     data, addr = sock.recvfrom(1024) # buffer size is 1024 bytes
     counter += 1
-    print("python - ", counter)
-    #print("received message: {}\tfrom: {}".format(data, addr))172.25.151.17
\ No newline at end of file
+    #print("python - ", counter)
+    if os.name == 'nt':
+        ip = data[0:4]
+        port = data[4:6]
+        data = data[6:]
+        ipstring = ''
+        for octet in ip:
+            ipstring = ipstring + str(octet) + "."
+        port = (port[0] << 8) + port[1]
+        print("IP: ", ipstring)
+        print("port: ", port)
+        print("data: ", data)
+    else:
+        print("received message: {}\tfrom: {}".format(data, addr))
```

Comment:
There's a warning that says there's no newline at the end of this file. Does that need to be fixed?

Reply:
not a deal breaker, but I fixed it regardless